Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -808,6 +806,87 @@ public static void ClearLocalSession()
         }
         #endregion
 
+        #region Remote Sessions
+        /// <summary>
+        /// Start a lease request for a remote session
+        /// If you want to let your local user sign in using another device then you use this method to get the lease code that the secondary device can use to authenticate.
+        /// While the process is ongoing, poll StartRemoteSession until the status is AUTHORIZED. When it is, a valid session to use locally will be in the response.
+        /// </summary>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        public static void LeaseRemoteSession(Action<LootLockerLeaseRemoteSessionResponse> onComplete)
+        {
+            if (!CheckInitialized(true))
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerLeaseRemoteSessionResponse>());
+                return;
+            }
+
+            LootLockerLeaseRemoteSessionRequest leaseRemoteSessionRequest = new LootLockerLeaseRemoteSessionRequest();
+
+            EndPointClass endPoint = LootLockerEndPoints.leaseRemoteSession;
+            LootLockerServerRequest.CallAPI(endPoint.endPoint, 
+                endPoint.httpMethod, 
+                LootLockerJson.SerializeObject(leaseRemoteSessionRequest), 
+                (serverResponse) => onComplete?.Invoke(LootLockerResponse.Deserialize<LootLockerLeaseRemoteSessionResponse>(serverResponse)), 
+                false);
+        }
+
+        /// <summary>
+        /// Create a new session authorized remotely from a previously constructed lease
+        /// If the session has not yet been remotely authorized, the response will contain only the status of the lease process
+        /// </summary>
+        /// <param name="remoteSessionLeaseResponse">TODO: Document</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        public static void StartRemoteSession(LootLockerLeaseRemoteSessionResponse remoteSessionLeaseResponse, Action<LootLockerStartRemoteSessionResponse> onComplete)
+        {
+            StartRemoteSession(remoteSessionLeaseResponse.code, remoteSessionLeaseResponse.nonce, onComplete);
+        }
+
+        /// <summary>
+        /// Create a new session authorized remotely from a previously constructed lease
+        /// If the session has not yet been remotely authorized, the response will contain only the status of the lease process
+        /// </summary>
+        /// <param name="leaseCode">The lease code returned in the LeaseRemoteSession response</param>
+        /// <param name="nonce">The nonce returned in the LeaseRemoteSession response</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        public static void StartRemoteSession(string leaseCode, string nonce, Action<LootLockerStartRemoteSessionResponse> onComplete)
```

Comment:
Maybe we should rename this to something like "PollAndStartRemoteSession", since calling it once does not guarantee that a session will be started.